applications/body_pose_estimation/export_yolo_pose_onnx.py

```
@@ -0,0 +1,104 @@
+    missing: list[str] = []
+    try:
+        import onnxslim  # noqa: F401
+    except ImportError:
+        missing.append("onnxslim>=0.1.71")
+    try:
+        import onnxruntime  # noqa: F401
+    except ImportError:
+        missing.append("onnxruntime")
+    if not missing:
+        return
+    subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)